securedrop_client/gui/widgets.py

```diff
@@ -1786,6 +1786,25 @@ def _on_reply_failure(self, message_id: str) -> None:
         if message_id == self.message_id:
             self._set_reply_state('FAILED')
 
+    @pyqtSlot(str, str, str)
+    def _update_text(self, source_id: str, message_id: str, text: str) -> None:
```

Comment:
I think we should discuss this change that you added - this is not necessary in order to fix #825 and I believe it complicates the code unnecessarily. The issue you refer to is already resovled in the `SendReplyJob` because it will know whether or not it gets a response back from the server. If it times out the job will always be readded to the queue and the next time around it will see that the reply was actually sent because it'll exist in the database.


This change doesn't break anything and everything else looks good. I did regression testing around replies and their different states. Since we want to see the nightly succeed before cutting a release tomorrow, I'm going to go ahead and merge and we can discuss this after standup.

Reply:
We don't need to discuss it. If you've fixed it elsewhere and it's superfluous it should be removed.